### PR TITLE
Update NotionalFinance.json

### PR DESCRIPTION
**Pull request by xiaona423**
Notional- zh

### CODE REVIEW

**Review by DarkGhost7 — APPROVED** [review]
lgtm @Bee926 could you tl check?

File: data/protocols/1/NotionalFinance.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     },
     "zh": {
       "name": "Notional Finance",
-      "description": "Notional Finance is a fixed-rate crypto lending and borrowing platform. It helps users to earn interest with certainty and minimize yield volatility."
+      "description": "Notional Finance 是一个固定利率借贷平台。帮助用户规避波动，赚取确定的收益。"
```
Comment (Bee926) [inline data/protocols/1/NotionalFinance.json:40]: change: 帮助用户规避波动，赚取确定的收益
to: 帮助用户赚取确定的收益并最小化收益率的波动

**Review by DarkGhost7 — APPROVED** [review]
lgtm

**Comment by Bee926** [comment]
done, good to go

**Pull request merged**